docs/01-Headless/01-Configuration.md

```diff
@@ -1,5 +1,26 @@
 # Configuration
 
-## TODO
+## Overview
 
-To be done, issue #392
+We call **Configuration** the AgnosUI system to set the default state of widgets deeply in the application tree.  
+This is usually referred as **context injection** and is handy to configure a part of a web application, without having to pass properties down the chain.
+
+You can check how each framework handles this need on their respective documentation: [setContext in Svelte](https://svelte.dev/docs/svelte#setcontext), [passing data deeply in React](https://react.dev/learn/passing-data-deeply-with-context) and [context injection in Angular](https://angular.io/guide/creating-injectable-service).
+
+The configuration is available out of the box as soon as widgets are created using the AgnosUI headless factories.  
+The headless packages also provide utilities for each framework to retrieve and update widget configuration.
+
+## In Action
+
+One of the most simplest usecases of configuration is setting a default config for multiple sibling widgets.  
+Let's see it in action, using the Bootstrap flavour of the **Alert** component:
+
+```sample
+{Configuration:configuration/basic:278}
+```
+
+The Configuration is however more used when passing default states to all children, as shown below:
```

Comment:
The Configuration is then used by all children with the modified state, as shown below:

?